server/utils/datetime_utils.py

```
@@ -206,17 +206,18 @@ def format_date_iso(date_val: str) -> Optional[str]:
-            # Resolve target timezone; fall back to UTC if conf.tz is missing/invalid
-            try:
-                target_tz = conf.tz if isinstance(conf.tz, datetime.tzinfo) else ZoneInfo(conf.tz)
-            except (ZoneInfoNotFoundError, ValueError, TypeError):
-                target_tz = datetime.UTC
-            dt = dt.astimezone(target_tz)
+        else:
+            dt = dt.astimezone(datetime.UTC)
+
+        try:
+            target_tz = conf.tz if isinstance(conf.tz, datetime.tzinfo) else ZoneInfo(str(conf.tz))
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)